### PR TITLE
Fixes #349 - Changed directory for unprocessible mails

### LINKED ISSUE #349: Update outdated path for unprocessible mails
https://docs.zammad.org/en/latest/admin/console/other-useful-commands.html#reprocess-unprocessable-mails

Still claims `tmp/unprocessable_mail/` to be the home of mails with issues.
However, this has changed to `var/spool/unprocessible_mail/` with Zammad 6.0.

### PR DESCRIPTION
**Pull request by ralf401**

### CODE REVIEW

**Comment by ralf401** [comment]
Reminder: backport to main

File: admin/console/other-useful-commands.rst [hunk 1]
```diff
@@ -18,8 +18,8 @@ Reprocess unprocessable mails
 
 When Zammad encounters a mail it cannot parse (e.g. due to a parser bug or a
 malformed message), it will store the mail in
-``tmp/unprocessable_mail/<ID>.eml``,  give up on attempting to parse the mail,
-and will warn on the monitoring page that there are unprocessed mails.
+``var/spool/unprocessible_mail/<ID>.eml``,  give up on attempting to parse the
```
Comment (mgruner) [inline admin/console/other-useful-commands.rst:21]: Please correct this to `var/spool/unprocessable_mail`, that's what the code uses.

**Review by mgruner — APPROVED** [review]
The folder name is wrong, please correct it.

`UNPROCESSABLE_MAIL_DIRECTORY = Rails.root.join('var/spool/unprocessable_mail')`

File: admin/console/other-useful-commands.rst (hunk 1 above)
Comment (ralf401) [reply admin/console/other-useful-commands.rst:21]: :see_no_evil: Ooops. Thank you for your hint!!

**Pull request merged**